pam/config/sgnl_pam.json

```
@@ -1,5 +1,5 @@
 {
-	"url": "https://access.sgnlapis.cloud/access/v1/evaluations",
+	"url": "{{insert SGNL URL}}",
```

```suggestion
	"url": "{{SGNL Access API Endpoint}}",
```
We should probably make the placeholder consistent across envoy and PAM